app/assets/stylesheets/member-facing/registration.scss

```diff
@@ -8,8 +8,29 @@
     margin-bottom: 20px;
   }
 
+  p {
+    padding-bottom: 10px;
```

Comment:
it's nested in my very own `registration` block. Still think a class would be better?